### DISCUSSION TITLE
Is Falcon Quantum Secure Signatures Supported By Bouncy Castle ?

### DISCUSSION THREAD

**Discussion by EAOE3** [bcgit/bc-java#1898] [Q&A]
I am looking for a library the support the Falcon signature algorithm and I'm wondering if bouncy castle already supports it.

**Comment by dghgit** [comment]
Yes we support it. Keep in mind it should be treated as experimental still. The official version FN-DSA is rumored to be coming out in draft in the next couple of months, it is likely when it does there will be some changes to the algorithm.
